interface/procedure_tools/labcorp/gen_hl7_order.inc.php

```diff
@@ -817,6 +820,10 @@ function send_hl7_order($ppid, $out)
         // Compute the target path/file name.
         $filename = $msgid . '.hl7';
         if ($pprow['orders_path']) {
+            if (!file_exists($pprow['orders_path'])) {
+                // attempt to make the directory
+                mkdir($pprow['orders_path']);
```

Comment:
If order_path contains file name this will create wrong path. But I think you've tested already.

Reply:
orders_path is just the directory, the filename gets added after.